lms/envs/common.py

```diff
@@ -361,7 +361,7 @@
     # There is also an advanced setting in the course module.  The
     # feature flag and the advanced setting must both be true for
     # a course to use saved grades.
-    'ENABLE_SUBSECTION_GRADES_SAVED': False,
+    'ENABLE_SUBSECTION_GRADES_SAVED': True,
```

Comment:
Should we be enabling this feature flag in test if we're going to be releasing this?